packages/core/src/app/init.test.ts

```diff
@@ -0,0 +1,30 @@
+import Koa from 'koa';
+import request from 'supertest';
+
+import { envVariablesSetUp } from '@/utils/test-utils';
+
+/**
+ * Need to mock env variables ahead
+ */
+
+envVariablesSetUp();
```

Comment:
can we move it to `jest.setup`?

Reply:
Use to add it as a `jest.setup`. But not confident whether we need to set the env variables for all the ut cases.  So decide to keep it in file for now. Move to jest.setup till the next requirement.

Reply:
ok